src/components/20-molecules/dropdown/react/story.jsx

```diff
@@ -2,32 +2,14 @@
 import { storiesOf } from '@storybook/html';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import AXADropdownReact from './AXADropdownReact';
+import DemoDropdownReact from './DemoDropdownReact';
 import { withMarkdown } from '../../../../../.storybook/addons/markdown';
 import Readme from '../README.md';
 
 storiesOf('Molecules/Dropdown/React', module)
   .addDecorator(withMarkdown(Readme))
   .add('Dropdown as React Component', () => {
     const div = document.createElement('div');
-    ReactDOM.render(
-      <AXADropdownReact
-        data-test-id="dropdown-react"
-        items={[
-          {
-            name: 'Please Select',
-            value: 'Please Select',
-            url: '#',
-            isSelected: true,
-            isInitialItem: true,
-          },
-          { name: 'Item 1', value: 'Item 1', url: '#', isSelected: false },
-          { name: 'Item 2', value: 'Item 2', url: '#', isSelected: false },
-          { name: 'Item 3', value: 'Item 3', url: '#', isSelected: false },
-        ]}
-        onAXAValueChange={value => console.log(`value changed ${value}`)}
-      />,
-      div
-    );
+    ReactDOM.render(<DemoDropdownReact />, div);
```

Comment:
Oh, now I see, that you did this on purpose. ö.ö

Would be worth a discussion, maybe you have a good reason to do that. :) Because all other stories and demos aren't handled like that.

Reply:
`git grep 'console.log' | grep story | wc -l` gives 12 occurrences, so not true.

Reply:
OK, I see your point and will see what I can do here.